pkg/services/live/push/push.go

```diff
@@ -19,7 +25,13 @@ var (
 )
 
 func init() {
-	registry.RegisterServiceWithPriority(&Gateway{}, registry.Low)
+	registry.RegisterServiceWithPriority(NewGateway(), registry.Low)
+}
+
+// CachedKey ...
+type CachedKey struct {
```

Comment:
Nit, but can we avoid exporting this?
```suggestion
// cachedKey ...
type cachedKey struct {
```

Reply:
Yeah, definitely 👍